internal/src/main/java/com/onewelcome/internal/util/AppVersionInfo.kt

```diff
@@ -0,0 +1,24 @@
+package com.onewelcome.internal.util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.res.stringResource
+import com.onewelcome.showcaseapp.BuildConfig
+import com.onewelcome.showcaseapp.R
+
+data class AppVersionInfo(
+  val version: String,
+  val omiSdkVersion: String
+)
+
+@Composable
+fun appVersionInfo(): AppVersionInfo {
```

Comment:
I checked and this method is being called each time the recomposition happens even thought there's `remember`. I guess this would need to be invoked somwhere higher in the composable hierarchy to be remembered correctly 🤔 I'm also thinking that `AppVersionInfo`, `OSVersionInfo` and `TestResultFileCreator` fit more into view model reposnsibilities. This will require some additonal coding for getting strings etc. but I feel like it would be a bit cleaner. Let me know what do you think

Reply:
ok, let's keep it this way and we can refactor in the future in case there's need for more than build config fields. But if I understand correctly we don't need the `remember` in those convenice functions, as the resources are being read everytime anyway 🤔